pkg/api/controllers/attachment_test.go

```diff
@@ -46,6 +47,84 @@ func init() {
 //                         Tests for volume attachment                          //
 ////////////////////////////////////////////////////////////////////////////////
 
+func TestCreateVolumeAttachment(t *testing.T) {
+	var jsonStr = []byte(`{
+		"id": "f2dda3d2-bf79-11e7-8665-f750b088f63e",
+		"name": "fake volume attachment",
+		"description": "fake volume attachment",
+		"hostId": "202964b5-8e73-46fd-b41b-a8e403f3c30b",
+		"volumeId": "bd5b12a8-a101-11e7-941e-d77981b584d8",
+		"attachMode": "ro"
+	}`)
+	var expectedJson = []byte(`{
+		"id": "f2dda3d2-bf79-11e7-8665-f750b088f63e",
+		"name": "fake volume attachment",
+		"description": "fake volume attachment",
+		"status": "creating",
+		"volumeId": "bd5b12a8-a101-11e7-941e-d77981b584d8",
+		"hostId": "202964b5-8e73-46fd-b41b-a8e403f3c30b",
+		"connectionInfo": {
+			"driverVolumeType": "iscsi",
+			"connectionData": {
+				"targetDiscovered": true,
+				"targetIqn": "iqn.2017-10.io.opensds:volume:00000001",
+				"targetPortal": "127.0.0.0.1:3260",
+				"discard": false
+			}
+		}
+	}`)
+	var expected model.VolumeAttachmentSpec
+	json.Unmarshal(expectedJson, &expected)
+	fmt.Println(expected)
```

Comment:
If this is for debugging, please remove it

Reply:
okay